includes/documents/class-wcpdf-invoice.php

```diff
@@ -321,23 +321,34 @@ public function init_settings() {
 					'id'					=> 'number_format',
 					'fields'				=> array(
 						'prefix'			=> array(
+							'label'	=> __( 'Prefix' , 'woocommerce-pdf-invoices-packing-slips' ),
 							'placeholder'	=> __( 'Prefix' , 'woocommerce-pdf-invoices-packing-slips' ),
```

Comment:
Now that you have added the labels you should get rid of the placeholder, it doesn't add anything (and looks odd)

Reply:
The double arrow (`=>`) could use a little formatting to get it back in line with the others. Also for the suffix and padding label.

Reply:
> Now that you have added the labels you should get rid of the placeholder, it doesn't add anything (and looks odd)

Fixed here: 2c55e5b

> The double arrow (=>) could use a little formatting to get it back in line with the others. Also for the suffix and padding label.

Fixed here: 9c5ec3f